src/lib/app.js

```diff
@@ -46,8 +46,8 @@ module.exports = class App {
       yield knex.knex.migrate.rollback(knex.config)
       yield knex.knex.migrate.latest(knex.config)
     }
-    yield this.notificationWorker.start()
-    yield this.timerWorker.start()
+    this.notificationWorker.start()
+    this.timerWorker.start()
```

Comment:
Why not wait for notificationWorker and timeWorker?  Doesn't the main event loop need these to be ready when it starts?